desk/app/metagrab.hoon

```diff
@@ -132,7 +133,19 @@
     ::TODO  extract message from response body somehow?
     ?:  (gte cod 500)
       [%500 ?~(dat `(scot %ud cod) `q.data.u.dat)]
-    [%400 `(scot %ud cod)]
+    ::  experimental bot-protection detection, which the client may
+    ::  want to know about so it can retry in a more advanced way.
+    ::
+    :-  %400
```

Comment:
I was seeing `401`'s as well. Could we make the heuristic to include this slightly more expansive?

Reply:
This branches handles all `4xx` responses from the target. We flatten those into "status: 400" responses from metagrab, in the same way that all 5xx responses get flattened to "status: 500" etc.

If you care about knowing the "full" original status code, that could in theory be arranged, but any heuristics you might want to do with that should probably be handled in-agent instead.

Reply:
ahh I see, sounds like we're covered here then. 

Having indication of the "true" original status code sounds nice, but I can't think of any immediate need that would necessitate a change. Maybe `404` as a signal to avoid attempting via non-metagrab means?